web/src/routes/auth/login/+page.svelte

```diff
@@ -17,9 +17,9 @@
     </p>
 
     <LoginForm
-      onSuccess={() => goto(AppRoute.PHOTOS, { invalidateAll: true })}
-      onFirstLogin={() => goto(AppRoute.AUTH_CHANGE_PASSWORD)}
-      onOnboarding={() => goto(AppRoute.AUTH_ONBOARDING)}
+      onSuccess={async () => await goto(AppRoute.PHOTOS, { invalidateAll: true })}
```

Comment:
The before and after look equivalent to me and actually the before might be slightly more performant and less code

Reply:
I found that without awaiting the function, it would hang at the login page instead of navigating to the route 🤔

Reply:
Maybe there's a race condition and you're just getting lucky/unlucky?

Reply:
> I found that without awaiting the function, it would hang at the login page instead of navigating to the route 🤔

FYI `() => <some promise>` does not mean it's not awaited. That's just syntactic sugar for `async () => {await <some promise>}` (and other things). So it should really not make a difference. In fact, I think we should always use `() => ` where possible and afaik @jrasm91 advocates for this as well, so not really sure why you approved it - probably just overlooked it.

Reply:
Hmm so there seems to be a different issue here